azure-functions/src/lib.rs

```diff
@@ -213,31 +213,55 @@ fn create_host_file(script_root: &Path, verbose: bool) {
     .unwrap_or_else(|_| panic!("Failed to create '{}'", host_json.display()));
 }
 
+/*
+ * Merge implementation with Serde, from:
+ * https://stackoverflow.com/a/47142105
+ */
+fn merge(a: &mut Value, b: Value) {
+    match (a, b) {
+        (a @ &mut Value::Object(_), Value::Object(b)) => {
+            let a = a.as_object_mut().unwrap();
+            for (k, v) in b {
+                merge(a.entry(k).or_insert(Value::Null), v);
+            }
+        }
+        (a, b) => *a = b,
+    }
+}
+
 fn create_local_settings_file(script_root: &Path, worker_dir: &Path, verbose: bool) {
     let settings = script_root.join("local.settings.json");
+    let mut local: Value;
+    let required_json = json!({
+        "Values" : {
+            "FUNCTIONS_WORKER_RUNTIME": "Rust",
+            "languageWorkers:workersDirectory": worker_dir.parent().unwrap()
+        }
+    });
+    /*
+     * If the settings exists, we need to insert the right workersDirectory setting
+     */
     if settings.exists() {
-        return;
-    }
-
-    if verbose {
-        println!("Creating local settings file '{}'.", settings.display());
-    }
+        let file = fs::File::open(&settings).expect("Could not read the local.settings.json file");
```

Comment:
I think it might be best to check for the existence of the file and if it does not exist, generate a complete one like we do now.  Thus existing projects that were generated without a `local.settings.json` still function when built with a newer `azure-functions` crate.

Reply:
I believe that case is covered below in the `else` statement here. If the file doesn't exist, the `local` and `required_json` are merged before write.

Reply:
You're right, I misread the diff. I missed the `if settings.exist()` call above.